modules/restore-replace-crashlooping-etcd-member.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-unhealthy-etcd-member.adoc
+
+[id="restore-replace-crashlooping-etcd-member_{context}"]
+= Replacing an unhealthy etcd member whose etcd Pod is crashlooping
+
+This procedure details the steps to replace an etcd member that is unhealthy because the etcd Pod is crashlooping.
+
+.Prerequisites
+
+* You have identified the unhealthy etcd member.
+* You have verified that the etcd Pod is crashlooping.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have taken an etcd backup.
++
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure, so that your cluster can be restored if you encounter any issues.
+====
+
+.Procedure
+
+. Stop the crashlooping etcd Pod.
+
+.. Debug the node that is crashlooping.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+----
+$ oc debug node/ip-10-0-131-183.ec2.internal <1>
+----
+<1> Replace this with the name of the unhealthy node.
+
+.. Change your root directory to the host:
++
+----
+sh-4.2# chroot /host
+----
+
+.. Move the existing etcd Pod file out of the Kubelet manifest directory:
```

Comment:
s/Kubelet/kubelet/
I've seen it both uppercase and lowercase in our docs, but k8s docs uses lowercase so that is my suggestion.

Reply:
You're right, I did just look that up the other day as well and agree that it's lowercase, but I had copied this from another procedure and missed it!